pages/community/translations.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: "Translations"
+description: "Translation Sites"
+canonical: "/community/translation"
+---
+
+# Translations
+The ReScript project is exclusively curated in English, but there are also community-driven projects for translating the docs into different languages.
+
+**Note:** Our documentation platform is still in the shaping process, and we don't have any bigger plans introducing i18n / translation features just yet. If you decide to translate our resources, please make sure to follow SEO best practises and most importantly, add [canonical](https://moz.com/learn/seo/canonicalization#:~:text=A%20canonical%20tag%20(aka%20%22rel,content%20appearing%20on%20multiple%20URLs.) references to point to the original source, otherwise we might get mixed results with SEO / google search. When in doubt, please reach out to us [in the forum](https://forum.rescript-lang.org).
```

Comment:
Hm, the link url I provided seems to break the markdown syntax, maybe let's try this one:

```suggestion
**Note:** Our documentation platform is still in the shaping process, and we don't have any bigger plans introducing i18n / translation features just yet. If you decide to translate our resources, please make sure to follow SEO best practises and most importantly, add [canonical](https://moz.com/learn/seo/canonicalization) references to point to the original source, otherwise we might get mixed results with SEO / google search. When in doubt, please reach out to us [in the forum](https://forum.rescript-lang.org).
```